### PR TITLE
Make tests pass on Travis CI

### PR DESCRIPTION
**Pull request by fgm**
https://www.drupal.org/node/2904149

### CODE REVIEW

File: src/Queue/Queue.php [hunk 1]
```diff
@@ -91,6 +79,7 @@ public function createItem($data) {
   public function numberOfItems() {
     // Retrieve information about the queue without modifying it.
     $queue_options = ['passive' => TRUE];
+    unset($this->queue);
```
Comment (phcorp) [inline src/Queue/Queue.php]: You may set this class defined property to null and let GC handle object deletion

File: README.md [hunk 2]
```diff
@@ -60,17 +60,17 @@ This is an example of how should looks like the `settings.php`:
...
+  'ssl' => [
+    'verify_peer_name' => false,
+    'verify_peer' => false,
+    'local_pk' => '~/.ssh/id_rsa',
+  ],
   'options' => [
-  'connection_timeout'    => 20,
-  'read_write_timeout'    => 20
+  'connection_timeout' => 20,
+  'read_write_timeout' => 20,
```
Comment (phcorp) [inline README.md]: Please indent

File: src/Queue/QueueFactory.php [hunk 3]
```diff
@@ -15,11 +16,19 @@ class QueueFactory {
 
   const SERVICE_NAME = 'queue.rabbitmq';
```
Comment (phcorp) [inline src/Queue/QueueFactory.php:16]: Not sure if service name should be put in a class constant as it is defined outside of this class (in `rabbitmq.services.yml`).

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,91 @@
+language: php
+
+php:
+  #- '7.0'
```
Comment (phcorp) [inline .travis.yml]: PHP 7.0 may be enabled or requirements updated?

File: src/Queue/Queue.php (hunk 1 above)
Comment (fgm) [reply src/Queue/Queue.php]: indeed.

File: README.md (hunk 2 above)
Comment (fgm) [reply README.md]: yep

File: src/Queue/QueueFactory.php (hunk 3 above)
Comment (fgm) [reply src/Queue/QueueFactory.php:16]: Easier to understand than a bare string, when reading code, IMO, so won't change.

File: .travis.yml (hunk 4 above)
Comment (fgm) [reply .travis.yml]: Indeed, I need to update the matrix.

**Pull request closed without merging**